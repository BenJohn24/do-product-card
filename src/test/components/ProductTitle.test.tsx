import { ProductTitle, ProductCard } from "../../components";
import { render, screen } from '@testing-library/react';
import { product1 } from "../data/products";



describe('ProductTitle', () => {
    test('debe de mostrar el componente correctamente con el titulo personalizado', () => {
   
      render(<ProductTitle title="Hola" className="custom-class"/>)

      console.log( );
    });

    test('debe de mostrar el component con el nombre del producto', () => {

      const { asFragment } = render(<ProductCard  product={ product1}>
        {
          () => (
            <ProductTitle />
          )
        }
        </ProductCard>
      
      )
      expect(asFragment()).toMatchSnapshot();
    })
    
})