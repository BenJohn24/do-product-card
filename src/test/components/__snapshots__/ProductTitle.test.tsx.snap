// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ProductTitle > debe de mostrar el component con el nombre del producto 1`] = `
<DocumentFragment>
  <div
    class="_productCard_133f98 undefined"
  >
    <span
      class="_productDescription_133f98 undefined"
    >
      Coffe Mug - Sing Imagen
    </span>
  </div>
</DocumentFragment>
`;
